no_hotplug removed in 3.17-rc6
this fixes compilation error due to that

patch by EmanueL Czirai

diff -pru5N -X /home/emacs/build/kernel/linuxgit/makepkg/linux-git/src/linux-git/Documentation/dontdiff -- archive_files.orig/common/lib/modules/fglrx/build_mod/kcl_acpi.c archive_files/common/lib/modules/fglrx/build_mod/kcl_acpi.c
--- archive_files.orig/common/lib/modules/fglrx/build_mod/kcl_acpi.c	2014-07-11 11:46:19.000000000 +0200
+++ archive_files/common/lib/modules/fglrx/build_mod/kcl_acpi.c	2014-09-21 09:50:35.974333364 +0200
@@ -838,11 +838,17 @@ static acpi_status KCL_ACPI_Slot_No_Hotp
    if(tdev != NULL)
    {
       device = (acpi_device_adr(tdev) >> 16) & 0xffff;
       if(PCI_SLOT(pdev->devfn) == device)
       {
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(3,17,0)
+         //as per this change: from 3.17.0-rc5 to 3.17.0-rc6
+         //https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=b29f83aa8b7018cdbe687dd57b239160e25889f0
+         pci_ignore_hotplug(pdev);
+#else
          tdev->flags.no_hotplug = true;
+#endif
       }
    }
 #endif
    return 0;
 }
